tutorials/polkadot-sdk/parachains/build-custom-pallet/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Pallet Tutorials
+description: TODO
+hide: 
+    - feedback
+template: index-page.html
+---
+
+# Tutorials for Building Custom Pallets with the Polkadot SDK
+
+## In This Section
+
```

Comment:
Where is this going to be taken care of? Is there a diff PR this is included in?

Reply:
yes, we are working on that with Luca at the moment, there is no PR yet, but it's on this branch: [0xlucca/update-tutorial-section](https://github.com/polkadot-developers/polkadot-docs/tree/0xlucca/update-tutorial-section)

Once we create that PR, we'll tag you so you can take a look at index pages :)